DQM/BeamMonitor/test/Alca_BeamMonitor_file.py

```diff
@@ -20,55 +47,48 @@
     )
 )
 #process.MessageLogger.statistics.append('cout')
-
 process.maxEvents = cms.untracked.PSet(
-    input = cms.untracked.int32(-1)
+    input = cms.untracked.int32(2000)
 )
 
-process.load("DQMServices.Core.DQM_cfg")
-process.load("DQMServices.Components.DQMEnvironment_cfi")
 
 process.load("DQM.BeamMonitor.AlcaBeamMonitor_cff")
+
+
 process.load("CondCore.DBCommon.CondDBSetup_cfi")
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+from Configuration.AlCa.GlobalTag import GlobalTag as gtCustomise
+process.GlobalTag = gtCustomise(process.GlobalTag, 'auto:run2_data', '')
+# you may need to set manually the GT in the line below
+#process.GlobalTag.globaltag = '100X_upgrade2018_realistic_v10'
+
+
+# DQM Live Environment
+process.load("DQM.Integration.config.environment_cfi")
+process.dqmEnv.subSystemFolder = 'BeamMonitor'
+process.dqmSaver.tag           = 'BeamMonitor'
+
+process.dqmEnvPixelLess = process.dqmEnv.clone()
+process.dqmEnvPixelLess.subSystemFolder = 'BeamMonitor_PixelLess'
+
 
-process.BeamSpotDBSource = cms.ESSource("PoolDBESSource",
-                                        process.CondDBSetup,
-                                        toGet = cms.VPSet(cms.PSet(
-    								   record = cms.string('BeamSpotObjectsRcd'),			        
-#    								   tag = cms.string('BeamSpotObjects_2009_LumiBased_v16_offline') 
-    								   tag = cms.string('BeamSpotObject_ByLumi') 
-    								  )
-						         ),								        
-                                         #connect = cms.string('frontier://cmsfrontier.cern.ch:8000/Frontier/CMS_COND_31X_BEAMSPOT')
-                                         connect = cms.string('sqlite_file:step4.db')
-                                         #connect = cms.string('oracle://cms_orcoff_prod/CMS_COND_31X_BEAMSPOT')
-                                         #connect = cms.string('frontier://PromptProd/CMS_COND_31X_BEAMSPOT')
-                                        )
-
-
-
-process.DQMStore.verbose = 0
-process.DQM.collectorHost = 'cmslpc08.fnal.gov'
-process.DQM.collectorPort = 9190
-process.dqmSaver.dirName = '.'
-process.dqmSaver.convention = 'Offline'
-process.dqmSaver.workflow = '/DQM/TkAlCalibration/ALCARECO'
-process.dqmEnv.subSystemFolder = 'AlcaBeamMonitor'
-process.dqmSaver.saveByRun = 1
-#process.dqmSaver.saveAtJobEnd = True
 
 #import RecoVertex.BeamSpotProducer.BeamSpotOnline_cfi
 #process.offlineBeamSpotForDQM = RecoVertex.BeamSpotProducer.BeamSpotOnline_cfi.onlineBeamSpotProducer.clone()
 
 # # summary
 process.options = cms.untracked.PSet(
-    wantSummary = cms.untracked.bool(True)
+    wantSummary = cms.untracked.bool(True),
+    numberOfThreads = cms.untracked.uint32(4),
+    numberOfStreams = cms.untracked.uint32 (4),
+    numberOfConcurrentLuminosityBlocks = cms.untracked.uint32(2)
+
     )
 
-process.pp = cms.Path(process.AlcaBeamMonitor+process.dqmSaver)
+process.pp = cms.Path(process.alcaBeamMonitor+process.dqmSaver)
```

Comment:
Is this a typo?

Reply:
I realize it is not, sorry

Reply:
no, it is intentional.
process.AlcaBeamMonitor is just running the EDAnalyzer
process.alcaBeamMonitor is a sequence which first runs a module needed to retrieve one of the many BeamSpot values and after runs the EDAnalyzer.